### DISCUSSION TITLE
NO_BLINK_LEV_IND doesn't seem to work

### DISCUSSION THREAD

**Discussion by classicrocker883** [mriscoc/Ender3V2S1#506] [General]
In file proui/dwin.cpp 
line ~640

>void _draw_ZOffsetIcon() {
  #if HAS_LEVELING
    #if NO_BLINK_LEV_IND
      static bool _leveling_active = false;
      if (_leveling_active != planner.leveling_active) {
        _leveling_active = planner.leveling_active;
        DWIN_Draw_Rectangle(1, HMI_data.Background_Color, 186, 415, 205, 436);
        DWINUI::Draw_Icon(_leveling_active ? ICON_SetZOffset : ICON_Zoffset, 187, 416);
      }

what is `NO_BLINK_LEV_IND`?
this part of the function appears to do nothing.

**Comment by mriscoc** [comment]
The definition of **NO_BLINK_LEV_IND** avoid leveling indicator background blinks
